cmonge/models/embedding.py

```diff
@@ -141,6 +149,7 @@ def __init__(
         checkpoint: bool,
         name: str,
         model_dir: str,
+        split_dose: bool = True,
```

Comment:
attribute not used within constructor. Even if it's needed, I would use `dose_split` consistently and not mix with `split_dose`